package.json

```diff
@@ -32,6 +32,7 @@
     "logo": "https://opencollective.com/react-native-elements/logo.txt"
   },
   "dependencies": {
+    "color": "^3.1.0",
```

Comment:
what does this packages provide us?

Reply:
It's a module to work with colors as strings and perform operations like lighten, darken that you'd normally have with sass etc. 

Used it here https://github.com/react-native-training/react-native-elements/pull/1540/files#diff-a0a6368880417eca5ad26c484e9ded7aR215